language-support/ts/daml-react/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "es2015"
+     ],
+    "jsx": "react",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "esModuleInterop": true,
+    "outDir": "lib/",
+    "module": "CommonJS",
+    "declaration": true,
+    "sourceMap": true
+    },
+  "files": ["index.ts"]
```

Comment:
This seems wrong. There are way more TypeScript files in here.

Reply:
this has already been fixed, thanks.